pkgs/cupertino_http/ios/cupertino_http.podspec

```diff
@@ -0,0 +1,29 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint cupertino_http.podspec` to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'cupertino_http'
+  s.version          = '0.0.1'
+  s.summary          = 'Flutter Foundation URL Loading System'
+  s.description      = <<-DESC
+  A Flutter plugin for accessing the Foundation URL Loading System.
+                       DESC
+  s.homepage         = 'https://github.com/dart-lang/http/tree/master/pkgs/cupertino_http'
+  s.license          = { :type => 'BSD', :file => '../LICENSE' }
+  s.author           = { 'Flutter Dev Team' => 'flutter-dev@googlegroups.com' }
```

Comment:
This is probably not the author you want.

Reply:
I set it to a placeholder for now.